build.sh

```diff
@@ -3,11 +3,17 @@ set -xeuo pipefail
 
 srcdir=$(pwd)
 
-# Init submodules
-ls -al .git mantle
-dnf -y install git
+# Work around https://github.com/coreos/coreos-assembler/issues/27
+if ! test -d .git; then
+    dnf -y install git
+    (git config --global user.email dummy@example.com
+     git init && git add . && git commit -a -m 'dummy commit'
+     git tag -m tag dummy-tag) >/dev/null
```

Comment:
This breaks the mantle `git describe` but oh well...

Reply:
Were we using that anywhere?

Reply:
https://github.com/coreos/mantle/blob/master/build#L13